src/canvas/static_canvas.class.ts

```diff
@@ -283,7 +281,24 @@ export class StaticCanvas<
 
   constructor(el: string | HTMLCanvasElement, options = {}) {
     super();
-    this._init(el, options);
+    this.renderAndReset = this.renderAndReset.bind(this);
+    this.requestRenderAll = this.requestRenderAll.bind(this);
```

Comment:
these were renamed,
see description

Reply:
Yes yes i have read everything i was in the middle of things and then i had to go to bed.
Probably going to let them both names.